packages/playground/website/src/components/layout/index.tsx

```diff
@@ -159,6 +160,10 @@ function Modals(blueprint: Blueprint) {
 	useEffect(() => {
 		addCrashListener(logger, (e) => {
 			const error = e as CustomEvent;
+			if (error.detail?.source) {
+				logErrorEvent(error.detail.source);
+			}
```

Comment:
While testing this I realized that request errors show only a white screen without any error messages.
I think that we can improve this, so I opened an issue https://github.com/WordPress/wordpress-playground/issues/1993

Reply:
Would it make sense to also log errors when the `source` is empty?